addons/crm/models/crm_lead.py

```diff
@@ -490,6 +490,11 @@ def _fields_view_get(self, view_id=None, view_type='form', toolbar=False, submen
             res['arch'] = self._fields_view_get_address(res['arch'])
         return res
 
+    @api.onchange('partner_id')
+    def onchange_partner_id(self):
```

Comment:
Why doing it in an onchange, and not in an editable computed field ? Is it intended ?